src/main/scala/net.manub.embeddedkafka/schemaregistry/streams/EmbeddedStreamsConfigImpl.scala

```diff
@@ -11,6 +11,6 @@ final class EmbeddedStreamsConfigImpl
 
   override def config(streamName: String, extraConfig: Map[String, AnyRef])(
       implicit kafkaConfig: EmbeddedKafkaConfig): Map[String, AnyRef] =
-    baseStreamConfig(streamName) ++ EmbeddedKafka.consumerConfigForSchemaRegistry ++ extraConfig
+    baseStreamConfig(streamName) ++ EmbeddedKafka.configForSchemaRegistry ++ extraConfig
```

Comment:
This is a non-BC change: I'd rather use `specificAvroReaderConfigForSchemaRegistry` instead, to keep it BC.